src/sampler.jl

```diff
@@ -62,12 +62,9 @@ function sample(
         if !(adaptor isa Adaptation.NoAdaptation)
             if i <= n_adapts
                 adapt!(adaptor, θs[i], αs[i])
-                # Finalize adapation
-                if i == n_adapts
-                    finalize!(adaptor)
-                    (verbose && !progress) && @info "Finished $n_adapts adapation steps" adaptor τ.integrator h.metric
-                end
+                i == n_adapts && finalize!(adaptor)
                 h, τ = update(h, τ, adaptor)
+                (i == n_adapts && verbose && !progress) && @info "Finished $n_adapts adapation steps" adaptor τ.integrator h.metric
```

Comment:
I change the order of `update` and finish-of-adapation info so that we indeed report the number after the last update.